FLEDGE_extended_PA_reporting.md

```diff
@@ -255,6 +256,82 @@ The `reserved.always` event-type is a special value that indicates a report shou
 matter whether a bid wins the auction or not. The browser will always trigger reporting for the
 `always` contributions.
 
+## Per-participant metrics
+
+More recent versions of Chrome offer some additional features, focused on measuring resource
+usage and whole-auction metrics rather than those specific to a particular function execution
+or the winning bid.
+
+First, the new `reserved.once` event-type is a special value that, for each sub-auction, selects a
+random invocation of `generateBid()` and of `scoreAd()` independently, and reports private
+aggregation contributions with that event only from those executions. (In case of a multi-seller
+auction, the top-level auction will have a single `scoreAd()` invocation selected as well).
+
+The event may not be used in `reportWin()` or `reportResult()`; since those already run at most
+once per auction level, `reserved.always` may be used instead.
+
+This feature is intended for reporting overall per-participant metrics only once rather than for
+every interest group participating. A number of new `baseValues` representing such values are
+available and described below, but it can also be useful with per-IG metrics which are not expected to
+vary much like `signals-fetch-time`, to sample them.
+
+While usage of `reserved.once` will be ignored by older versions, newly added `baseValues` will not
+be, so the calls to `contributeToHistogramOnEvent()` should be individually wrapped in `try/catch`.
+That is also encouraged in general since `contributeToHistogramOnEvent()` is specified to throw
+on permission policy violations.
+
+Users using this are strongly encouraged to report their metrics during the beginning of their
+scripts, since if the script hits a per script time out before asking to report them nothing will
+get sent, which can result in inaccuracy, especially for `percent-scripts-timeout`.
+
+### Per-participant base values.
+
+The newly added base values are as following:
+* `participating-ig-count`: number of interest groups that got a chance to participate in this
+  (sub)auction, i.e. they had registered ads, and if no timeouts were to hit, their `generateBid()`
```

Comment:
It's supposed to be a hypothetical?

Reply:
Ah, I see. Perhaps...?

i.e. they had registered ads and their `generateBid()` was queued/scheduled to run.

(timeouts aren't the only thing that could deep six their participation. the auction could be aborted, &c. )

Reply:
I don't really want to talk about queueing/scheduling since it's a bit of an implementation detail --- e.g. in our implementation generateBid isn't actually scheduled to run until all of relevant promises resolve. 
Anyway, tried to rephrase it to be more concrete.

(Also aborts presently completely drop PA ... which I suspect we will need to change...)